WalletWasabi.Fluent/Controls/TagControl.axaml

```diff
@@ -20,7 +20,7 @@
     <Setter Property="ClipToBounds" Value="False" />
     <Setter Property="Template">
       <ControlTemplate>
-        <Panel Name="PART_TagPanel" Margin="0,-3,0,0">
```

Comment:
Changing to this:

 `<Panel Name="PART_TagPanel" Margin="0,3,2,0">` 
 
 fixes the problem reported [here](https://github.com/zkSNACKs/WalletWasabi/pull/7168#pullrequestreview-880882047)

Reply:
thanks! i adjusted it some more since it didnt looked correct here on mac